scripts/azure-pipelines/build_linux.bash

```diff
@@ -7,7 +7,21 @@
 
 ver=$(python scripts/mumble-version.py)
 
-qmake -recursive CONFIG+="release tests warnings-as-errors" DEFINES+="MUMBLE_VERSION=${ver}"
+# Use the Qt kit installed through PPA
+source /opt/qt*/bin/qt*-env.sh || true
```

Comment:
I'm not sure it's a good idea to hide a failure of sourcing that file. Isn't tit more or less required for the rest to work?

Reply:
Uhm now we are almost back from where we started xD

I could imagine that the `source` command has problems because it might get fed with more than 1 argument (due to the `*` operator) and one of these arguments I suppose is not sourceable. Thus `source` returns `1`.
I'd rather fix the source of the error than having such a workaround in there as this would also hide the case in whcih `source` would fail altogether